Base/res/html/misc/private-element-test.html

```diff
@@ -0,0 +1,59 @@
+<html>
+<body>
+    <span>If passed, the text below should say: "Passed" And the iframe should have loaded</span>
+    <div id="result">Starting up</div>
+    <script>
+        class Base {
+            constructor(o) {
+                return o;
+            }
+        }
+
+        class Stamper extends Base {
+            #x = 10;
+            static hasX(o) { return #x in o; }
+        }
+
+        let failed = false;
+        function status(text, fail=false) {
+            console.log('Status', text, fail, failed);
+            const output = document.getElementById("result");
+            if (fail) {
+                output.innerHTML += "Fail: " + text + "<br />";
+                output.style.color = "red";
+                failed = true;
+            } else if (!failed) {
+                output.innerText = text;
+                output.style.color = "blue";
```

Comment:
This doesn't draw the page meaning you have to resize the browser to get it to update, any workaround for that?

Reply:
.innerText is more unreliable than .innerHTML (not sure why). I find building the nodes in JS then using something like replaceChild works best though.

Reply:
I've switched to `.innerHTML` I don't think this will be such an important test.